playbooks/prepare_test_host.yml

```diff
@@ -30,7 +30,7 @@
       version: "{{ csm_version }}"
       force: yes
       depth: 1
-  - name: Install ansible roles
+  - name: Install ansible roles  # noqa 301 305
```

Comment:
move this up, before "Ensure required packages" task

Reply:
Castling done